tests/__fixtures__/server/cert.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Could you please move all the cert files into a new `ssl/` folder inside the `__fixtures/server/` folder, and rename them to kebab case (`cert-with-pass.key`, `cert-pass`, etc.)?

Thanks!

Reply:
Done